nef/About/AboutView.swift

```diff
@@ -0,0 +1,89 @@
+//  Copyright © 2019 The nef Authors.
+
+import SwiftUI
+
+struct AboutView: View {
+    private let version: String
+    private let browser: Browser
+    
+    init(version: String, browser: Browser) {
+        self.version = version
+        self.browser = browser
+    }
+    
+    @State var option: Bool = true
+    var body: some View {
+        VStack(spacing: 16) {
+            nefView
+            SeparatorView(height: 16, color: .purple)
+            
+            InstallStepView(image: NefImage.preferences, opacity: 0.85, description: i18n.installPreferences)
+                .onTapGesture(perform: openPreferences)
+            InstallStepView(image: NefImage.extensions, opacity: 0.9, description: i18n.installExtensions)
+                .onTapGesture(perform: openPreferences)
+            FixedToggle(title: "nef", description: i18n.nefExtension, isOn: true)
+            
+            Spacer()
+            
+            githubView.padding(8)
+            Text(i18n.copyright)
+                .foregroundColor(.init(white: 0.7))
+            
+        }.frame(maxWidth: .infinity, maxHeight: .infinity)
+         .padding(16)
+    }
+    
+    // MARK: internal subviews
+    private var nefView: some View {
+        HStack(spacing: 6) {
+            NefImage.logo
+                .resizable().frame(width: 50, height: 50)
+                .foregroundColor(.init(white: 0.4))
+            nefVersionView
+        }
+    }
+    
+    private var nefVersionView: some View {
+        VStack {
+            Text("nef")
+                .font(.system(size: 30)).fontWeight(.bold)
+            Text("\(version)")
+                .font(.system(size: 10))
+                .foregroundColor(.gray)
+        }
+    }
+    
+    private var githubView: some View {
+        Text(i18n.github)
+            .foregroundColor(.blue)
+            .onTapGesture(perform: openGithub)
+    }
+    
+    // MARK: actions
+    private func openPreferences() {
+        browser.open(url: "x-apple.systempreferences:com.apple.preferences")
+    }
+    
+    private func openGithub() {
+        browser.open(url: Constants.githubURL)
+    }
+    
+    // MARK: Constants
+    enum NefImage {
+        static let logo = Image("nef-favicon")
+        static let preferences = Image("system-preferences")
+        static let extensions = Image("system-extensions")
+    }
+    
+    enum Constants {
+        static let githubURL = "https://github.com/bow-swift/nef#-features"
+    }
+    
+    enum i18n {
+        static let copyright = NSLocalizedString("Copyright © 2019 The nef authors", comment: "")
```

Comment:
I think `NSLocalizedString` is no longer needed when working with SwiftUI. Check [this](https://nathfreder.dev/localizing-your-swiftui-app/).

Reply:
uh! good point, I did not know it will go auto - I will change it

Reply:
I have reviewed it and it only works if we use 'harcoded key' directly in the constructor, for example:

`Toggle('preferences_key', isOn: $state.value)`
`Text('preferences_key')`
`Image('preferences_image')`

But if we want for example reuse the same key in different elements, and prevent to use 'hardcoded' strings in the code, we can not do it:

`Toggle(Const.preferencesKey, isOn: $state.value)`
`Text(Const.preferencesKey)`

Const.PreferencesKey = 'preferences_key'

So, I think it is better to continue using `NSLocalizedString` in favour of reusability. Wdut @truizlop ?

Reply:
On another hand, I have localized the plugin (en|es)

Reply:
Ok, it's good to know that.